static/styles/_variables.scss

```diff
@@ -17,3 +17,10 @@ $tablet-width: 768px;
 $medium-screen: 900px;
 $large-screen: 1024px;
 $x-large-screen: 1200px;
+
+// blog page
+$side-nav-bg-color: #f9f9f9;
```

Comment:
go ahead and move these to `static/styles/partials/_colors.scss`

Also, do you mind changing the rgb colors to hex colors for consistency?

@meowwwls do you have any suggestions for the naming of these colors?

Reply:
I actually think we can do away with all of these new colors.

I was wondering if the blue used for "read me" should be changed to a color already used on the site, such a `$dark-teal`? This blue is not used anywhere else and it's quite different from the rest of the colors used. 

I would also get rid of `$side-nav-color` and where it is used, use `$lead-gray` which is close enough (I suggested the previous color before I saw the ones available already).

Where `$side-nav-bg-color` is used, I would instead use `lighten($lt-grey, 25%)`. This is an accessible combination which also gives consistency to the whole site.

Reply:
Thank for clarifying... 
I changed to `lighten($lt-grey, 25%)` to `llighten($lead-gray, 25%)` as `lt-grey` isn't defined...

FYI, I attached screen shoots
<img width="567" alt="on hover" src="https://user-images.githubusercontent.com/17010408/66702804-39c27c80-ed29-11e9-8951-8d10b5e32c7f.png"> 
*on hover*
<img width="554" alt="normal color" src="https://user-images.githubusercontent.com/17010408/66702805-3a5b1300-ed29-11e9-8255-59cac3c35143.png"> 
*normal color*
<img width="129" alt="side nav" src="https://user-images.githubusercontent.com/17010408/66702806-3a5b1300-ed29-11e9-93ca-83abbaac39f0.png"> 
*side nav*

Side nav code
```
.blog-side-nav-school-non-selected {
  color: $lead-gray;
  background-color: lighten($lead-gray, 25%);
}

.blog-side-nav-school-selected {
  color: lighten($lead-gray, 25%);
  background-color: $lead-gray;
}
```
I feel the side nav can further be improved... @meowwwls pls drop your cmt...

Reply:
Sorry, it should be `lighten($light-grey, 25%)`, not $lt-grey like I thought it was. $lead-grey is too dark for this purpose.